Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -84,6 +84,22 @@ void OnDestroy()
             }
         }
 
+        void OnDisable()
+        {
+            if (keywordRecognizer != null)
+            {
+                StopKeywordRecognizer();
+            }
+        }
+
+        void OnEnable()
+        {
+            if (keywordRecognizer != null && RecognizerStart == RecognizerStartBehavior.AutoStart)
```

Comment:
According to https://docs.unity3d.com/Manual/ExecutionOrder.html, OnEnable will be called before Start. So you're right, the OnEnable will encounter a null keywordRecogizer, but Start should run right after (since this is the first time the script is active) and cause the expected behavior.